omod/src/main/webapp/template/localHeader.jsp

```diff
@@ -16,3 +16,5 @@
 <openmrs:htmlInclude file="/moduleResources/openhmis/cashier/js/init.js" />
 <openmrs:htmlInclude file="/moduleResources/openhmis/inventory/js/init.js" />
 <openmrs:htmlInclude file="/moduleResources/openhmis/backboneforms/js/curl.js" />
+<openmrs:htmlInclude file="/scripts/jquery/jquery.min.js" />
```

Comment:
I don't think this is the correct jquery file. You need to reference the jquery files in the backbone resources (like the reference above).